package.json

```diff
@@ -18,18 +18,18 @@
   },
   "devDependencies": {
     "@automattic/nx-composer": "^0.1.0",
-    "@nrwl/cli": "^13.3.4",
-    "@nrwl/linter": "^13.3.4",
-    "@nrwl/devkit": "^13.1.4",
+    "@nrwl/cli": "^13.4.2",
```

Comment:
E2E PRs should not make any changes to the monorepo dependencies. The changes to this file should be reverted.

Reply:
The dependency bump came after a `pnpm update` to install the new dev dependencies of the `e2e-core-tests` package. I did this because CI failed entirely with ` ERR_PNPM_OUTDATED_LOCKFILE  Cannot install with "frozen-lockfile" because pnpm-lock.yaml is not up-to-date with packages/js/e2e-core-tests/package.json`.

Would it be preferable that I undo the dependency bumps and do just a `npm update` instead, inside the e2e-core-tests package? I thought that the `ERR_PNPM_OUTDATED_LOCKFILE` was monorepo asking for a `pnpm update` due to the centralized dependecies

Reply:
`pnpm update` reverted, `pnpm install --no-frozen-lockfile` instead